libraries/asset-registrar/src/ev-registry.ts

```diff
@@ -59,7 +59,7 @@ export class EvRegistry {
     console.log('[EV REGISTRY] user does not exist');
     const user = await this._contract.signer.getAddress();
     const { r, s, v } = await this._getSignature(user);
-    await this._contract.addUser(user, v, r, s);
+    await this._contract.addUser(user, v, r, s, { gasLimit: 1000000 });
```

Comment:
I think a comment to explain why the below is there would be helpful. Also maybe worth explaining why `1000000` was chosen
```suggestion
    /** 
     * gasLimit added because of "unpredictable gas limit" message received when executing on Volta.
     * Likely this could be removed once Volta is upgraded to EIP-1559 and London hardfork
      */
    await this._contract.addUser(user, v, r, s, { gasLimit: 1000000 });
```

Reply:
the choice of 1000000 is arbitrary, I just set it to a large number.